debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleStreamingChangeEventSourceMetrics.java

```diff
@@ -615,14 +632,21 @@ public void setLastCommitDuration(Duration lastDuration) {
         }
     }
 
-    public void setTimeDifference(long timeDifference) {
-        this.timeDifference.set(timeDifference);
+    public void calculateTimeDifference(OffsetDateTime databaseSystemTime) {
```

Comment:
Could you add a JavaDoc comment describing the purpose of this method? "time difference between what" would be a great thing to capture, so we still do know in 3 months from now.